ufo2fontir/src/source.rs

```diff
@@ -939,15 +1008,15 @@ impl Work<Context, WorkError> for KerningWork {
                         return None;
                     }
                     let group_name = GroupName::from(name);
-                    if !kerning.groups.contains_key(&group_name) {
-                        debug!("'{name}' references non-existent group {group_name}; ignored");
+                    let Some(group_name) = old_to_new_group_name.get(&group_name) else {
+                        debug!("'{name}' is not a valid group name; ignored");
```

Comment:
better to issue a warning since we are effectively dropping a kerning pair value here, better to alert the user so they fix their group names

Reply:
Updated several similar debugs to warn